### ISSUE TITLE
Ramp up or tutorial

### ISSUE THREAD

**Issue by quozl** [sugarlabs/turtleart-activity#49]
A study found that the activity is rarely used to construct a working block set, so a ramp up or tutorial may be helpful for situations where no teacher or peer instruction is received.

[S. Nogry & P. Varly (2018) Everyday Laptop Use by Children in a Southern
Country: A Mixed-Method Approach, Journal of Research on Technology in Education, 50:1, 18-33,
DOI: 10.1080/15391523.2017.1388200](https://www.tandfonline.com/doi/pdf/10.1080/15391523.2017.1388200?needAccess=true)

**Comment by quozl** [comment]
Yes, plenty of Wiki and examples already.  The guide looks good.  Research of Madagascar shows the ramping examples were not enough, and that embedding a tutorial into the application may be better.  Implode has an embedded tutorial.

**Comment by Dimi20cen** [comment]
Would adding a link for turtleart's wiki in the READEME be a step in the good direction?

**Comment by chimosky** [comment]
> Would adding a link for turtleart's wiki in the READEME be a step in the good direction?

Not really as the README would most likely only be seen by developers but adding more examples to the [samples](https://github.com/sugarlabs/turtleart-activity/tree/master/samples) in the activity is a great way to go. 

And including some tutorial like in Implode activity as quozl has pointed out would be great.

**Comment by Dimi20cen** [comment]
Got it, will work on that

**Comment by walterbender** [comment]
We have lots of "example" projects and (in the case of Music Blocks) we have lots of lesson plans. No need to invent a new one for hollow lines. Let's find a fun example and build from that.

**Comment by GAMinsect** [comment]
@walterbender ok will think of something.

Moreover, would you still want me to develop a implode like tutorial, or is this fun example just fine?

**Comment by GAMinsect** [comment]
@walterbender ok will try to do it

**Comment by GAMinsect** [comment]
A squiral is a square spiral right?
